benchmarks/benchmark7.ipynb

```diff
@@ -298,7 +298,7 @@
     "\n",
     "For all of these simulations, verify that the time step is small enough that any temporal error is much smaller that the total error. This can be accomplished by decreasing the time step until it has minimal effect on the error. Ensure that at least three simulation results have $L_2$ errors in the range $[5\\times10^{-3}, 1\\times10^{-4}]$, attempting to cover as much of that range as possible/practical. This maximum and minimum errors in the range roughly represent a poorly resolved simulation and a very well-resolved simulation.\n",
     "\n",
-    "For at least three simulations that have $L_2$ errors in the range $[5\\times10^{-3}, 1\\times10^{-4}]$, save the effective mesh size and $L_2$ error in a CSV or JSON file. Upload this file to the PFHub website as a 2D data set with the effective mesh size as the x-axis column and the $L_2$ error as the y-axis column. Calculate the effective element size as the square root of the area of the finest part of the mesh for nonuniform meshes. For irregular meshes with continous distributions of element sizes, approximate the effective mesh size as the average of the square root of the area of the smallest 5% of the elements.\n",
+    "For at least three simulations that have $L_2$ errors in the range $[5\\times10^{-3}, 1\\times10^{-4}]$, save the effective mesh size and $L_2$ error in a CSV or JSON file. [Upload this file to the PFHub website](https://pages.nist.gov/pfhub/simulations/upload_form/) as a 2D data set with the effective mesh size as the x-axis column and the $L_2$ error as the y-axis column. Calculate the effective element size as the square root of the area of the finest part of the mesh for nonuniform meshes. For irregular meshes with continous distributions of element sizes, approximate the effective mesh size as the average of the square root of the area of the smallest 5% of the elements.\n",
```

Comment:
Not sure why we ask benchmarkers to filter their data, why we're requesting
"effective mesh size," or why we ask people to upload CSV files directly to
PFHub instead of an archive. Simplify:

    Save the effective element size $h$ and $L_2$ error for each simulation.
    [Archive this data](https://github.com/usnistgov/pfhub/issues/491) in a
    CSV or JSON file, using one column (or key) each for $h$ and $L_2$. 
    Calculate the effective element size as the square root of the area of
    the finest part of the mesh for nonuniform meshes. For irregular meshes
    with continuous distributions of element sizes, approximate the effective
    element size as the average of the square root of the area of the smallest
    5% of the elements. Then [submit your results on the PFHub website]
    (https://pages.nist.gov/pfhub/simulations/upload_form/).

Reply:
@tkphd I fixed the "pi"s as requested. I modified the archive/upload section along the lines requested, and made similar modifications to related sections further on.

I kept sections describing what to list as the x axis and y axis while submitting the results, because I assume we want that standardized.